src/app/(main)/my-voyage/[teamId]/tech-stack/components/TechStackCard.tsx

```diff
@@ -98,12 +188,17 @@ export default function TechStackCard({ title, data }: TechStackCardProps) {
                 key={element.id}
               >
                 {isEditing === element.id && (
```

Comment:
I think this needs better naming. isEditing makes me think it's a boolean, but that doesn't seem to be the case here.

Reply:
Yeah. I originally had it as a boolean. Maybe 'editItemId' would be a better name?

Reply:
yup that seems better